importer/api/__init__.py

```diff
@@ -142,6 +144,7 @@ def import_children(course, element, parent):
         content_xml=etree.tostring(element),
         mpath=mpath,
     )
+    target = "/static/"
```

Comment:
no, leave it there as it is: if it starts to be an issue we will change it later